documentation/modules/serving/pages/_partials/deploy-knative-resources.adoc

```diff
@@ -95,7 +95,7 @@ kn service create prime-generator \
 [.console-input]
 [source,bash,subs="+macros,+attributes"]
 ----
-kn service update prime-generator \
+kn service create prime-generator \
```

Comment:
There's a cleanup instruction just before this command in the docs. So, update doesn't work.